t/29_re_flags.t

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use Test::More 0.47;
+
+use Perl::MinimumVersion;
+
+my %examples=(
+    q{use re "/xx"; } => '5.026',
+    q{use re "/mxsx"; } => '5.026',
+    q{use re "debug"; } => '5.005',
+    q{use re; } => '5.005',
+    q{use re qw(taint /xx); } => '5.026',
+    q{use re qw(taint /x); } => '5.005',
```

Comment:
This needs to be `'5.014'`.

Reply:
I've fixed that with my latest commit on this PR df825e6 "Bump PPIx::Regexp and fix `re flags` min versions".